alacritty/src/event.rs

```diff
@@ -1176,13 +1176,13 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                 match event {
                     WindowEvent::CloseRequested => self.ctx.terminal.exit(),
                     WindowEvent::Resized(size) => {
-                        // Minimizing the window sends a Resize event with zero width and
-                        // height. But there's no need to ever actually resize to this.
-                        // ConPTY has issues when resizing down to zero size and back.
-                        //
                         // The user could also resize the window to have zero height
```

Comment:
Having `also` here doesn't make any sense anymore since it's the first sentence now.

Suggesting this:

```
// Ignore resize events to zero in any dimension, to avoid issues with Winit and the ConPTY.
//A 0x0 resize will also occur when the window is minimized on Windows.
```

I think this should cover all the information we currently have but provide it in a little more structured manner.